test/blackbox-tests/melange-playground/compile-ml-warning-alert.t

```diff
@@ -0,0 +1,53 @@
+  $ cat > input.js <<EOF
+  > require(process.env.DUNE_SOURCEROOT + '/_build/default/bin/jsoo_main.bc.js');
+  > require(process.env.DUNE_SOURCEROOT + '/_build/default/bin/melange-cmijs.js');
+  > console.log(ocaml.compileML("let t = Js.Vector.length"));
+  > EOF
+
+  $ node input.js
+  {
+    js_code: '// Generated by Melange\n' +
+      '\n' +
+      '\n' +
+      'function t(prim) {\n' +
+      '  return prim.length;\n' +
+      '}\n' +
+      '\n' +
+      'export {\n' +
+      '  t ,\n' +
+      '}\n' +
+      '/* No side effect */\n',
+    warnings: [
+      {
+        js_warning_error_msg: 'Line 1, 8:\n' +
+          '  Warning: deprecated module Js.Vector\n' +
+          'Use Belt.Array instead',
+        row: 0,
+        column: 8,
+        endRow: 0,
+        endColumn: 24,
+        text: 'module Js.Vector\nUse Belt.Array instead',
+        type: 'warning'
```

Comment:
this is not a warning though :( it's an alert

Is there a way we could report these as alerts (with the alert name too?)